lib/driver.js

```diff
@@ -30,7 +30,7 @@ import AsyncLock from 'async-lock';
 import path from 'path';
 import IDB from 'appium-idb';
 import DEVICE_CONNECTIONS_FACTORY from './device-connections-factory';
-
+import { parseServerArgs } from '@appium/base-driver';
```

Comment:
This change will stop this driver from being used by Appium v1. Is it what we want?

Reply:
why would it do that? we're just importing a single function from a different package. this single function does not depend on anything appium 2.0 related.